docusaurus.config.js

```diff
@@ -408,25 +408,25 @@ const config = {
             ],
             to: "/main/build/modules/upgrade/",
           },
-          {
-            from: ["/main/modules/capability", "/main/ecosystem"],
-            to: "/main/build/modules/",
-          },
+          // {
```

Comment:
this needed to be commented out to build

Reply:
https://github.com/cosmos/cosmos-sdk/pull/17881 will fix this